modules/ROOT/pages/errors/gql-errors/index.adoc

```diff
@@ -140,107 +140,107 @@ Status description:: error: data exception - unsupported value. The provided val
 
 === xref:errors/gql-errors/22N10.adoc[22N10]
 
-Status description:: error: data exception - invalid pre-parser option value. Invalid pre-parser option, specified `{ $input }` is not valid for option `{ $option }`. Valid options are: `{ $optionList }`.
+Status description:: error: data exception - invalid pre-parser option value. Invalid pre-parser option, specified `{ <<input>> }` is not valid for option `{ <<option>> }`. Valid options are: `{ <<optionList>> }`.
 
 === xref:errors/gql-errors/22N11.adoc[22N11]
 
-Status description:: error: data exception - invalid argument. Invalid argument: cannot process `{ $input }`.
+Status description:: error: data exception - invalid argument. Invalid argument: cannot process `{ <<input>> }`.
 
 === xref:errors/gql-errors/22N12.adoc[22N12]
 
-Status description:: error: data exception - invalid date, time, or datetime format. Invalid argument: cannot process `{ $input }`.
+Status description:: error: data exception - invalid date, time, or datetime format. Invalid argument: cannot process `{ <<input>> }`.
 
 === xref:errors/gql-errors/22N13.adoc[22N13]
 
 Status description:: error: data exception - invalid time zone. Specified time zones must include a date component.
 
 === xref:errors/gql-errors/22N14.adoc[22N14]
 
-Status description:: error: data exception - invalid temporal value combination. Cannot select both `{ $temporal1 }` and `{ $temporal2 }`.
+Status description:: error: data exception - invalid temporal value combination. Cannot select both `{ <<temporal>>1 }` and `{ $temporal2 }`.
 
 === xref:errors/gql-errors/22N15.adoc[22N15]
 
-Status description:: error: data exception - invalid temporal component. Cannot read the specified `{ $component }` component from `{ $temporal }`.
+Status description:: error: data exception - invalid temporal component. Cannot read the specified `{ <<component>> }` component from `{ <<temporal>> }`.
 
 === xref:errors/gql-errors/22N18.adoc[22N18]
 
-Status description:: error: data exception - incomplete spatial value. A `{ $crs }` POINT must contain `{ $mapKeyList }`.
+Status description:: error: data exception - incomplete spatial value. A `{ <<crs>> }` POINT must contain `{ <<mapKeyList>> }`.
 
 === xref:errors/gql-errors/22N19.adoc[22N19]
 
 Status description:: error: data exception - invalid spatial value. A `POINT` must contain either 'x' and 'y', or 'latitude' and 'longitude'.
 
 === xref:errors/gql-errors/22N20.adoc[22N20]
 
-Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ $dim1 }D` coordinate reference system (CRS) and `{ $value }` coordinates. Use the equivalent `{ $dim2 }D` coordinate reference system instead.
+Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ <<dim>>1 }` coordinate reference system (CRS) and `{ <<value>> }` coordinates. Use the equivalent `{ $dim2 }` coordinate reference system instead.
```

Comment:
```suggestion
Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ <<dim>>1 }`D coordinate reference system (CRS) and `{ <<value>> }` coordinates. Use the equivalent `{ $dim2 }`D coordinate reference system instead.
```

Looks like the D:s have fallen out, the parameter is e.g. 2 or 3 so it becomes 2D or 3D